src/core_modules/capture-core/components/Pages/Search/SearchPage.component.js

```diff
@@ -91,88 +153,42 @@ const Index = ({
-                                            // Therefore we are returning the group title and the
-                                            // SingleSelectOption in an array.
-                                            [
-                                                <SingleSelectOption
-                                                    value={trackedEntityTypeId}
-                                                    label={trackedEntityTypeName}
-                                                />,
-                                                tePrograms.map(({ programName, programId }) =>
-                                                    (<SingleSelectOption value={programId} label={programName} />)),
-                                                <div className={classes.divider} key={trackedEntityTypeId}>
-                                                    <hr />
-                                                </div>,
-                                            ],
-                                        ),
-                                    [
-                                        classes.divider,
-                                        trackedEntityTypesWithCorrelatedPrograms,
-                                    ])
-                                }
-                            </SingleSelect>
-                        </div>
-                    </div>
-                </Section>
+
+                <SearchSelection
+                    trackedEntityTypesWithCorrelatedPrograms={trackedEntityTypesWithCorrelatedPrograms}
+                    setSelected={setSelected}
+                    selectedOption={selectedOption}
+                />
+
+                <SearchForm
+                    selectedOptionId={selectedOption.value}
+                    availableSearchOptions={availableSearchOptions}
+                />
 
                 {
-                    selectedOption.value && availableSearchOptions[selectedOption.value].searchGroups
-                        .filter(searchGroup => searchGroup.unique)
-                        .map(({ searchForm, formId }) => {
-                            const name = searchForm.getElements()[0].formName;
-                            return (
-                                <Section
-                                    className={classes.searchDomainSelectorSection}
-                                    header={
-                                        <SectionHeaderSimple
-                                            containerStyle={{ paddingLeft: 8, borderBottom: '1px solid #ECEFF1' }}
-                                            title={i18n.t('Search {{name}}', { name })}
-                                        />
-                                    }
+                    searchStatus === searchPageStatus.NO_RESULTS &&
+                    <Modal position="middle">
+                        <ModalTitle>Empty results</ModalTitle>
```

Comment:
We will have to make a PR where we correct the text in the Search page as a whole. When I am developing this feature most of the people who are responsible for the text were away.